distributed/diagnostics/plugin.py

```diff
@@ -190,7 +190,7 @@ def add_worker(self, scheduler: Scheduler, worker: str) -> None | Awaitable[None
         """
 
     def remove_worker(
-        self, scheduler: Scheduler, worker: str, *, stimulus_id: str, **kwargs: Any
+        self, scheduler: Scheduler, worker: str, name: str, *, stimulus_id: str, **kwargs: Any
```

Comment:
Is there a reason why this is being added as a position argument rather than a keyword argument? 

```suggestion
        self, scheduler: Scheduler, worker: str, *, name: str, stimulus_id: str, **kwargs: Any
```

Reply:
Thanks, that makes sense!